sources/packages/web/src/components/aest/institution/modals/ApproveDenyDesignationModal.vue

```diff
@@ -0,0 +1,176 @@
+<template>
```

Comment:
Taking a second look at this component, it actually seems to be 2, one is a simple deny modal and the other is the more complex one for approval.
We are really far into all UI changes, do we have a common component for approval/deny with a simple note?
If yes, the deny component here can be the generic one, right?